### ISSUE TITLE
Can I read the  structure symbol information of S7-300 Data Blocks (SUBBLK.DBT) from a S7-Project?

### ISSUE THREAD

**Issue by mut9bu** [dotnetprojects/DotNetSiemensPLCToolBoxLibrary#183]
I want to read from existing S7-Projects (not from the PLC) the Data Block symbols & structure information.  I know that they are included in the SUBBLK.DBT file of the project, but I cannot decode the file correctly. Is it possible with your library?

**Comment by jogibear9988** [comment]
yes

**Comment by mut9bu** [comment]
Thanks a lot. I have installed the nugget and made some trials but couldn't find ow to read the general symbol list & the DB structures and symbols in SUBBLK.DBT. Is there a sample project for that? I have looked to the samples but couldn't bu sure which sample I should follow.  
To summarise What I want is: 1) I want to get list of the Structures of all DB's with the starting byte addresses from a project ,so that the user can read an elemnt in a DB from the PLC with its symbolic name (I will combine symbolic name with the absolute DB addres in my code)

2) I want to get out the symbol table list, so that the user can read out the I,Q,M,T,C from the PLC with the symbol name. (I will combine the symbol names and the absolute addres in my code acc to the symbol list)
